tests/unit/connect.c

```diff
@@ -185,6 +182,9 @@ int main(int argc, char *argv[])
     drizzle_close(con);
   }
 
+  int cxt = 1;
+  drizzle_set_log_fn(con, log_fn_callback, (void*)&cxt);
+  drizzle_set_verbose(con, DRIZZLE_VERBOSE_DEBUG);
```

Comment:
Did you mean to change this from `DRIZZLE_VERBOSE_INFO` to `DRIZZLE_VERBOSE_DEBUG` as well?

Reply:
Yes. Know it looks a bit counter intuitive wrt to the ci message. 
The idea is that logging only is enabled for the last call to `drizzle_quit` but with DEBUG level. I will rephrase the ci msg to make this clear